### DISCUSSION TITLE
Trying to build a terrain-only map with the --buildmap option.

### DISCUSSION THREAD

**Discussion by ViridianGames** [exult/exult#312] [General]
I've built Exult 1.8.0 and I'm trying to build a map that only has the terrain tiles - no terrain objects like trees or rocks.  I'm launching Exult with the --buildmap option to do it, but so far all of my attempts to exclude objects have resulted in black PNGs.

I'm poking at the source code to see if I can figure this out but if anyone has any suggestions, I'd be very grateful.

**Comment by ViridianGames** [comment]
Okay, I figured it out.  I altered paint_map() in gamerend.cc to return before drawing the chunk's objects.

**Comment by DominusExult** [comment]
can you post what you changed or create a pull request with a sensible buildmap flag to paint these?

**Reply by ViridianGames** [reply]
Certainly.  I should have a pull request for you this weekend or so.
